.github/CIP-TEMPLATE.md

```diff
@@ -46,7 +49,7 @@ It must also explain how the proposal affects the backward compatibility of exis
 <!-- Describes what are the acceptance criteria whereby a proposal becomes 'Active' -->
 
 ### Implementation Plan
-<!-- A plan to meet those criteria. Or `N/A` if not applicable. -->
+<!-- A plan to meet those criteria or `N/A` if an implementation plan is not applicable. -->
 
 ## Copyright
 <!-- The CIP must be explicitly licensed under acceptable copyright terms. -->
```

Comment:
should we add?:

`This CIP is licensed under [CC-BY-4.0][].`

Would make it slightly easier for authors and also ensures the `.` 😆.

Reply:
Technically yes, but at best it would violate the principle of a template if they intended to use the Apache license, and at worst it would have a different license in the header than in the footer.

In fact I believe the several Apache vs. CC license mismatches we have in remediated (#389) CIPs are from the time the license markup _was_ in an earlier version of the template.

Reply:
good spot, we shouldnt then!

I am just trying to make it slightly easier for authors and ourselves, as often the `Copyright` section is neglected to be filled in .

Reply:
> it would violate the principle of a template if they intended to use the Apache license

Well the template includes `CC-BY-4.0` in the header by default 😆

What about, in the `Copyright` section:

```
<!-- This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode). -->
<!-- This CIP is licensed under [Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0). -->
```

and we remove the potentially confusing markdown references part at the bottom?

Reply:
sure; let's do it that way (added some instruction as well, to try to reduce even more repetitive editing): https://github.com/cardano-foundation/CIPs/pull/730/commits/61974c20b3cac8c90837a4244c47591c4fc38def

I am also now wondering why the `Copyright` field or section aren't in the CPS template.  I was around for that big remodel but don't recall seeing why at least CC licensing isn't recommended since even the simplest license have a standard open source liability waiver that might protect interested parties in the case of some ambitious CPS. (cc @KtorZ @michaelpj)